services/provider/proto/provider.proto

```diff
@@ -84,12 +84,18 @@ message KubeObject {
 message RbdDriverRequirements {
     // topologyDomainLables are the list of topology that needs to be set in nodeplugin
     repeated string topologyDomainLables = 1;
+    // hostNetwork setting for controller plugin
+    optional bool CtrlPluginHostNetwork = 2;
```

Comment:
seeing around, the fields are camelCased and messages are PascalCased, let's follow the same.

Reply:
Updated